content/en/blog/_posts/2024-09-20-kubeadm-v1beta4.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: blog
+title: 'Kubernetes v1.31: kubeadm v1beta4'
+date: 2024-08-23
+slug: kubernetes-1-31-kubeadm-v1beta4
+author: >
+   Paco Xu (DaoCloud)
+---
+
+The [`kubeadm`](/docs/reference/setup-tools/kubeadm/) configuration
+([v1beta4](/docs/reference/config-api/kubeadm-config.v1beta4/))
+is released with Kubernetes v1.31. v1beta3 is still supported,
+but is marked as deprecated. It will be removed after a minimum
+of 3 Kubernetes releases.
```

Comment:
How about this?
```suggestion
As part of the Kubernetes v1.31 release, [`kubeadm`](/docs/reference/setup-tools/kubeadm/) is
adopting a new ([v1beta4](/docs/reference/config-api/kubeadm-config.v1beta4/)) version of
its configuration file format. Configuration in the previous v1beta3 format is now formally
deprecated, which means it's supported but you should migrate to v1beta4 and stop using
the deprecated format.1
Support for v1beta3 configuration it will be removed after a minimum of 3 Kubernetes minor releases.
```